src/Utils.js

```diff
@@ -25,11 +25,16 @@ export function queryKey(Components) {
   var names = [];
```

Comment:
I'd change `names` to `ids` now that we aren't using the component name.

Reply:
Thanks, fixed!